README.md

```diff
@@ -1,11 +1,29 @@
-# Express Project Skeleton
+# CodeX
 
-Use this project skeleton as a starting point for structuring your app. Things to note
-* Sequelize configuration has not yet been added -- you will need to set that up yourself
-* You may find yourself wanting to use javascript -- js files can be added in `public/javascripts` and should be appended to the Pug templates as needed
-* CSS files can go in `public/stylesheets` and also will need to be added to Pug templates
+Website: [https://codexappaa.herokuapp.com/](https://codexappaa.herokuapp.com/)
 
-## Demo user login
+## Technologies Used
 
-- Email: demo@demo.com
-- Password: password
+- JavaScript
+- Express
+- PUG
+- CSS
+
+## Features
+
+- Website home page with stories
+- User registration / login (with a demo account available)
+  - An authenticated home page with recommended stories and stories of users you follow
+- Speicifc story pages to view the story written by an author
+  - A comments section, where authenticated users can add comments to a story
+  - Like/unlike feature on the story and its comments that update live
+  - Follow/unfollow feature that updates live
+- A story creation page allowing authenciated users to post new stories
```

Comment:
Indentation seems a little off here?

Reply:
I had those indented as I felt like they were related to the parent list item. I can make them all the same level if you think that makes more sense.